packages/base64/test/bench-main.js

```diff
@@ -28,13 +32,32 @@ const data = encodeBase64(string);
   }
   const end = Date.now();
   const duration = end - start;
-  console.log(
+  log(
     'encodes',
     (operations * string.length) / duration,
     'characters per millisecond',
   );
 }
 
+{
+  const start = Date.now();
```

Comment:
I wish we used a slightly more precise `performance.now()` if available.

Reply:
Yeah, that bugged me too. Done.